app/src/main/java/com/peacecorps/pcsa/support_services/SupportServicesFragment.java

```diff
@@ -19,7 +19,7 @@
  * Support Services, main fragment
  *
  * @author rohan
- * @since 2016-07-08
+ * @since 2016-07-15
```

Comment:
It is okay to have the date you created the class as @since. Since only you are working on this, there is no need to change it to the day committed.